### ISSUE TITLE
Outline color needs to meet 3.1 color contrast ratio

### ISSUE THREAD

**Issue by kkoryaka** [Office-of-Digital-Services/California-State-Web-Template-Development#152]
Focus indicator outline does not meet contrast against the background. Make it a different color that passes 3:1

<img width="528" alt="Image" src="https://github.com/user-attachments/assets/300dd6dc-f44b-4afa-804d-2b71b35ebc80" />
